### ISSUE TITLE
make cli update even easier

### ISSUE THREAD

**Issue by mave99a** [ArcBlock/forge-cli#163]
Small improvement, since we prompt user run `npm install -g @arcblock/forge-cli` to update cli, why don't provide a sub command to run that npm for user? e.g. `forge update-cli` 

```
$ forge version

ℹ Latest forge-cli version is v0.37.1, your local version v0.36.8
ℹ Please upgrade with npm install -g @arcblock/forge-cli
```

**Comment by wangshijun** [comment]
Perhaps we should just ask user to decide whether he/she wants to upgrade, then just run `npm install -g @arcblock/forge-cli` internally.

**Comment by mave99a** [comment]
yes. agree

**Comment by wangshijun** [comment]
resolved in #177
